src/iop/liquify.c

```diff
@@ -3324,17 +3324,19 @@ void gui_init (dt_iop_module_t *module)
 
   gtk_box_pack_start(GTK_BOX(module->widget), hbox, TRUE, TRUE, 0);
 
-  dt_liquify_layers[DT_LIQUIFY_LAYER_PATH].hint           = _("ctrl-click to add node\nright click to remove path");
-  dt_liquify_layers[DT_LIQUIFY_LAYER_CENTERPOINT].hint    = _("click and drag to move - click : linear or feathered\n"
-                                                              "ctrl-click : autosmooth, cusp, smooth, symmetrical"
+  dt_liquify_layers[DT_LIQUIFY_LAYER_PATH].hint           = _("ctrl-click to add node\n"
+                                                              "right click to remove path\n"
+                                                              "ctrl-alt-click to toggle line/curve");
```

Comment:
OK. While I'm there, maybe replace `to` with `:` to gain a few characters and for consistency with most other strings. That'd give:

```
"ctrl-click: add node, right click: remove path\n"
"ctrl-alt-click: toggle line/curve"
```